main.tex

```diff
@@ -142,27 +139,28 @@ \section{The Software}
 a common framework that uses the Python \package{requests} package to perform
 HTTP requests to communicate with web services.
 
-For new development, there is a \texttt{template}  that lays out the basic
-framework of any new module.  All modules have a single core
+For new service development, there is a \texttt{template module} that lays
+out the basic framework of any new module.  All modules have a single core
 \texttt{class} that has some number of associated \texttt{query\_*} methods.
-The most common query methods are \texttt{query\_region}, which usually provide
-a ``cone search'' functionality, i.e., they search for data within a circularly
-symmetric region projected on the sky.
-
-An example using the SIMBAD interface is shown below (see
-\url{http://astroquery.readthedocs.io/en/latest/simbad/simbad.html}):
+The most common query methods are \texttt{query\_region}, which usually
+provide a ``cone search'' functionality, i.e., they search for data within a
+circularly symmetric region projected on the sky. The returned results of
+the queries then are returned in an \astropypkg
+\citep{Astropy-Collaboration2018, Astropy-Collaboration2013}
+\texttt{Table}\footnote{\url{http://docs.astropy.org/en/stable/table/}}.
+
+An example using the SIMBAD interface is shown below:\footnote{see
+\url{http://astroquery.readthedocs.io/en/latest/simbad/simbad.html}}
 \begin{lstlisting}[caption=Query SIMBAD for a region around M81]
 from astroquery.simbad import Simbad
 result_table = Simbad.query_region("m81")
 \end{lstlisting}
 In this example, \texttt{Simbad} is an instance of the
 \texttt{astroquery.simbad.SimbadClass} class.
-The returned result, stored in the variable \texttt{result\_table},
-is an \astropypkg table\footnote{\url{http://docs.astropy.org/en/stable/table/}}.
```

Comment:
Why is this sentence removed?  It's important to specify what is returned

Reply:
Oh, I see, you just moved it up.  That's OK

Reply:
yes, I think it belong to the description above as it hasn't said anything about the preferred return objects.